src/modules/output.js

```diff
@@ -0,0 +1,24 @@
+import { aplication } from "./variables";
+
+const showRefresh = (e) => {
```

Comment:
- [ ] As suggested by the previous code reviewer and as written in the project requirements, I kindly suggest that you ensure that your scores are updated only on the click of the refresh button. Currently immediately I submit, the scores are updated and they are only supposed to be updated when I click the refresh button. 
- [ ] The submit button should save the score while the refresh button gets the score and updates the page.

<img width="746" alt="image" src="https://user-images.githubusercontent.com/52934128/197030451-ab85c8d1-3af7-4d7c-ae93-0d0a7f7189e8.png">

Reply:
@topeogunleye Can you help me please?

Reply:
Sorry @luccamicale. I'm busy. Can you please contact your teammates for help.

Reply:
@luccamicale Here's an example pls.

## EXAMPLE
```

submitButton.addEventListener('click', async (event) => {
  event.preventDefault();
  const user = document.querySelector('#name').value;
  const score = Number(document.querySelector('#score').value);
  const newScore = {
    user,
    score,
  };

  game.addScore(newScore);

  //  clear input fields
  document.querySelector('#name').value = '';
  document.querySelector('#score').value = '';
});


const renderScores = async () => {
  const arrayOfScores = await game.getScore();
  leaderboard(arrayOfScores);
};

// render onClick of refresh button
document
  .querySelector('#refresh-button')
  .addEventListener('click', renderScores);

// render on page load
document.addEventListener('DOMContentLoaded', renderScores);
```